aiofreepybox/api/ftp.py

```diff
@@ -0,0 +1,23 @@
+class Ftp:
+
+    def __init__(self, access):
+        self._access = access
+
+    ftp_configuration_schema = {
+        'allowAnonymous': False,
+        'allowAnonymousWrite': False,
+        'enabled': False,
+        'password': ''
```

Comment:
The `FtpConfig` described [here](https://dev.freebox.fr/sdk/os/ftp/) is a bit different, has it changed in the new API version?

Reply:
'allow_remote_access' was added in v4, i don't know about other changes, but anyway, this kinds of represents the minimum set of parameters to make it work. Here you can set parameters as you like, just make sure you're ready when you set 'enabled' to True.

Reply:
I just tried out, here is the config I get (which matches the documentation I linked):
`{'enabled': False, 'allow_anonymous_write': False, 'remote_domain': 'xxxxx', 'port_data': 45678, 'weak_password': False, 'allow_remote_access': False, 'port_ctrl': 12345, 'allow_anonymous': False}`

Reply:
Fixed